pandas/core/internals.py

```diff
@@ -2614,6 +2587,9 @@ def __init__(self, values, placement, ndim=2, dtype=None):
         super(DatetimeTZBlock, self).__init__(values, placement=placement,
                                               ndim=ndim)
 
+    # override NonConsolidatableMixIn implementation of get_values
```

Comment:
remove this for now and revert get_values from the Block.

Reply:
So just keep the unnecessary implementation currently in DatetimeTZBlock?

Reply:
if you want to get this merged. yes. I really don't like this change the way it is. if you want to refactor into something more palable be my guest. but this PR would be on hold until then. so I would simply separate this (and to be honest it would have to be a convincing change).

Reply:
Will do, am just curious because usually you like removing duplicate code.

Reply:
yes, but in this case you are doing a completely orthognal change, which is already pretty spaghetti like, so a re-factor is in order. and separate that these changes.